src/app/leverage/layout.tsx

```diff
@@ -0,0 +1,20 @@
+import { Flex } from '@chakra-ui/react'
+
+import { Footer } from '@/components/footer'
+import { Header } from '@/components/header'
+
+import { Providers } from '../providers'
```

Comment:
Similar question here - can we use the 'Lite' version and reduce how much we're loading?